system-int-test/test/cmr/system_int_test/ingest/collection_update_test.clj

```diff
@@ -902,4 +901,59 @@
 
         "Updating an instrument  that is referenced by a granule by humanized alias back to its original value is invalid."
         ["p2" "i2" "GPS" "s1"]
-        ["Collection Child Instrument [GPS RECEIVERS] is referenced by existing granules, cannot be removed. Found 1 granules."]))))
+        ["Collection Child Instrument [gps receivers] is referenced by existing granules, cannot be removed. Found 1 granules."]))))
+
+(deftest collection-update-case-insensitivity-test
+  (let [a1 (data-umm-c/additional-attribute {:Name "STRING" :DataType "STRING"})
```

Comment:
I might not be reading this right, but it seems like you have a bunch of uppercase strings. You probably want some mixed case strings as well.